config/jest/client.other.config.js

```diff
@@ -0,0 +1,100 @@
+const fs = require("fs");
+const path = require("path");
+
+const d3Pkgs = [
+  "d3",
+  "d3-array",
+  "d3-axis",
+  "d3-brush",
+  "d3-chord",
+  "d3-color",
+  "d3-contour",
+  "d3-delaunay",
+  "d3-dispatch",
+  "d3-drag",
+  "d3-dsv",
+  "d3-ease",
+  "d3-fetch",
+  "d3-force",
+  "d3-format",
+  "d3-geo",
+  "d3-hierarchy",
+  "d3-interpolate",
+  "d3-path",
+  "d3-polygon",
+  "d3-quadtree",
+  "d3-random",
+  "d3-scale",
+  "d3-scale-chromatic",
+  "d3-selection",
+  "d3-shape",
+  "d3-time",
+  "d3-time-format",
+  "d3-timer",
+  "d3-transition",
+  "d3-zoom",
+];
+
+const dir = "./src/core/client";
+const dirItems = fs.readdirSync(dir);
+let roots = [];
+
+const excludedRoots = ["src/core/client/stream", "src/core/client/admin"];
+
+for (const item of dirItems) {
+  const fullPath = path.join(dir, item);
+  const stats = fs.statSync(fullPath);
+  if (stats.isDirectory() && !excludedRoots.includes(fullPath)) {
+    roots.push(fullPath);
+  }
+}
+
+roots = roots.map((r) => `<rootDir>/${r}`);
+
+module.exports = {
+  displayName: "client",
```

Comment:
Do we want to have `client:other` as the `displayName` here to match how the stream tests display name is configured?

Reply:
fixed